Tasks/AzureRmWebAppDeploymentV4/postDeploymentScript/deleteLogFile.cmd

```diff
@@ -1,21 +1,21 @@
 @echo off
 
-DEL "mainCmdFile_%1.cmd" 
+DEL "mainCmdFile.cmd" 
```

Comment:
should we need to pass the absolute file path here?

Reply:
not required as we are running delete_log_file script from the folder in which mainCmdFile resides